recipes/cargo-c/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set version = "0.9.24" %}
+
+package:
+  name: cargo-c
+  version: {{ version }}
+
+source:
+  url: https://github.com/lu-zero/cargo-c/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 32f2f5c802c01c51cf93471fcf876d0cc68edbc31d22005b9f07e4549d5b98b1
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+    - pkg-config
+  host:
+    - libcurl
+    - openssl
+    - zlib
```

```suggestion
requirements:
  build:
    - {{ compiler('c') }}
    - {{ compiler('rust') }}
    - cargo-bundle-licenses
    - pkg-config
  host:
    - libcurl
    - openssl
    - zlib
  run:
    - pkg-config
```
It wasn't clear to me from upstream docs whether pkg-config is needed at runtime or just compiling this package.